pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,75 @@
+# JSON Schema documentation contributing guide 
+
+This guide describes the workflow to contribute to the JSON Schema technical documentation. If you would like to contribute to other aspects of the JSON Schema website, like writing blog articles or publishing a case study, please read the website's [Contributing Guide](/website/CONTRIBUTING.md) for more details. 
+
+## Welcome
+Thank you for helping us improve our documentation.
+
+This guide is intended for use by documentation contributors and will help you get started with the basics of our working processes.
```

Comment:
It will be great if we add a message for code contributors ending here and redirecting them to the general guidelines for code contribution guidelines